Problem compiling with latest Rust on Windows

Reported by @kingdonb:

> I had to use an older version of Rust to compile Bartholomew because of some dependency that had been using a now deprecated and removed feature. It was a challenge to figure out from the error message on Rust 1.70.0.

https://github.com/fermyon/spin-vscode/issues/34

The error is:

```
error[E0793]: reference to packed field is unaligned
    --> C:\Users\ivan\.cargo\registry\src\index.crates.io-6f17d22bba15001f\ntapi-0.3.7\src\ntexapi.rs:2783:52
     |
2783 |         *tick_count.QuadPart_mut() = read_volatile(&(*USER_SHARED_DATA).u.TickCountQuad);
     |                                                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
     |
     = note: fields of packed structs are not properly aligned, and creating a misaligned reference is undefined behavior (even if that reference is never dereferenced)
     = help: copy the field contents to a local variable, or replace the reference with a raw pointer and use `read_unaligned`/`write_unaligned` (loads and stores via `*p` must be properly aligned even when using raw pointers)

error[E0793]: reference to packed field is unaligned
    --> C:\Users\ivan\.cargo\registry\src\index.crates.io-6f17d22bba15001f\ntapi-0.3.7\src\ntexapi.rs:2807:25
     |
2807 |         ((read_volatile(&(*USER_SHARED_DATA).u.TickCountQuad)
     |                         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
     |
     = note: fields of packed structs are not properly aligned, and creating a misaligned reference is undefined behavior (even if that reference is never dereferenced)
     = help: copy the field contents to a local variable, or replace the reference with a raw pointer and use `read_unaligned`/`write_unaligned` (loads and stores via `*p` must be properly aligned even when using raw pointers)

For more information about this error, try `rustc --explain E0793`.
error: could not compile `ntapi` (lib) due to 2 previous errors
```

There's a more recent version of ntapi but it's an indirect dependency (via mio I think).

--- a/bart/Cargo.toml
+++ b/bart/Cargo.toml
@@ -10,7 +10,7 @@ bartholomew = { path = "../", default-features = false }
 chrono = { version = "0.4.19", features = ["serde"] }
 colorful = "0.2.1"
 structopt = "0.3.22"
-tokio = { version = "1.8.1", features = [ "full" ] }
+tokio = { version = "1.29", features = [ "full" ] }
 toml = "0.5.8"
 tracing = { version = "0.1.26", features = [ "log" ] }
 tracing-futures = "0.2.5"

